editor/editor_properties_array_dict.cpp

```diff
@@ -819,6 +823,16 @@ void EditorPropertyArray::_reorder_button_up() {
 		array.call("remove_at", reorder_slot.index);
 		array.call("insert", reorder_to_index, value_to_move);
 
+		bool last_folding = object->editor_is_section_unfolded(object->get_property_name_for_index(reorder_slot.index));
+		for (int i = reorder_to_index; i != reorder_slot.index; i += (reorder_slot.index < reorder_to_index ? -1 : 1)) {
```

Comment:
The last part can be written as `SIGN(reorder_slot.index - reorder_to_index)` and you could put it in a variable, so it's not evaluated with every iteration.